src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/DefaultPageLoader.cs

```diff
@@ -33,22 +34,20 @@ public class DefaultPageLoader : IPageLoader
             _logger = logger;
         }
 
-        public Type Load(PageActionDescriptor actionDescriptor)
+        public CompiledPageActionDescriptor Load(PageActionDescriptor actionDescriptor)
```

Comment:
Can you file a follow up work item to add tests for this method + cleaning up the resx string here.